<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1559</width>
    <height>1332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>1091</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1071</width>
       <height>131</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금(d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>새 행</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>1071</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>등락률</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>1101</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>자동 선정 종목 리스트</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1081</width>
       <height>241</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>주문유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>호가구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>가격</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BDR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상태</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>770</y>
      <width>1091</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>실시간 체결현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1071</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>주문상태</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>미체결수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문시간</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>계좌</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1559</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
